ksqldb-common/src/test/java/io/confluent/ksql/function/UdfIndexTest.java

```diff
@@ -562,35 +563,6 @@ public void shouldFindNonVarargWithPartialNullValues() {
     assertThat(fun.name(), equalTo(EXPECTED));
   }
 
-  @Test
-  public void shouldChooseFirstAddedWithNullValues() {
-    // Given:
-    givenFunctions(
-        function(EXPECTED, false, STRING),
-        function(OTHER, false, INT)
-    );
-
-    // When:
-    final KsqlScalarFunction fun = udfIndex.getFunction(Collections.singletonList(null));
-
-    // Then:
-    assertThat(fun.name(), equalTo(EXPECTED));
-  }
-
-  @Test
-  public void shouldFindVarargWithNullValues() {
-    // Given:
-    givenFunctions(
-        function(EXPECTED, true, STRING_VARARGS)
-    );
-
-    // When:
-    final KsqlScalarFunction fun = udfIndex.getFunction(Arrays.asList(new SqlArgument[]{null}));
-
-    // Then:
-    assertThat(fun.name(), equalTo(EXPECTED));
-  }
```

Comment:
I'll need to figure out why this is failing, it's not immediately obvious to me.